schemas/json/theme.json

```diff
@@ -364,7 +364,8 @@
 							"items": {
 								"type": "string"
 							},
-							"default": [ "px", "em", "rem", "vh", "vw", "%" ]
+							"default": [ "px", "em", "rem", "vh", "vw", "%" ],
+							"minItems": 1
```

Comment:
Curious.. where is this value used? Does it need to be in the theme.json schema?

The schema is, I thought, more for validation and documentation, not to house arbitrary vars.

Reply:
`minItems` is a valid json schema property to indicate that the `units` array must have at least 1 entry. It is then used by an IDE to indicate that an empty array is invalid:
https://user-images.githubusercontent.com/3629020/284096504-bce929ea-f119-4017-bebd-c2634115f66d.png

That was my understanding of it anyway.

Reply:
https://json-schema.org/understanding-json-schema/reference/array#length

Reply:
Oh! [TIL](https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-validation-00#rfc.section.6.4.2)

Withdraw everything I said 😄 

Thanks for the explainer!